packages/shared/lib/utils/utils.ts

```diff
@@ -371,3 +376,8 @@ export function getConnectionConfig(queryParams: any): Record<string, string> {
     const arr = Object.entries(queryParams).filter(([_, v]) => typeof v === 'string'); // Filter strings
     return Object.fromEntries(arr) as Record<string, string>;
 }
+
+export function packageJsonFile(): PackageJson {
+    const localPath = process.env['SERVER_RUN_MODE'] === 'DOCKERIZED' ? 'packages/shared/package.json' : '../shared/package.json';
```

Comment:
would using require.resolve work for both case. ie: `require.resolve('@nangohq/shared/package.json')` ?

Reply:
```require is not defined``` is the error I get when attempting to do that.